recipes/widgetsnbextension/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.0.0" %}
+
+package:
+    name: widgetsnbextension
+    version: {{ version }}
+
+source:
+    fn: widgetsnbextension-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/41/98/e2424df4a1222df72451bddc4471ec15482cc8c125203ffcf1923173e6a6/widgetsnbextension-{{ version }}.tar.gz
```

Comment:
Try this with the URL.

`https://pypi.io/packages/source/w/widgetsnbextension/widgetsnbextension-{{ version }}.tar.gz`

See this [comment]( https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package#comment-27243225 ) in this [issue]( https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package ).